paddle/fluid/framework/ir/mkldnn/conv_activation_mkldnn_fuse_pass.cc

```diff
@@ -138,7 +137,7 @@ void ConvActivationMkldnnFusePass::FuseConvConcatAct(
     for (auto node : concat_inputs) {
       OpDesc* conv_op = node->inputs[0]->Op();
       if (conv_op->Type() == "conv2d") {
-        conv_op->SetType("fused_conv2d");
+        ConvertToFusedOp(conv_op);
```

Comment:
You can remove `if` statement because this check will be done in `ConvertToFusedOp` function.